bin/insertMissingAlerts.ts

```diff
@@ -0,0 +1,19 @@
+import '../src/config';
+import createOrGetConnection from '../src/db';
+import { Alerts, User } from '../src/entity';
+
+export const insertMissingAlerts = async () => {
+  const con = await createOrGetConnection();
+  const usersQuery = con
+    .getRepository(User)
+    .createQueryBuilder()
+    .select('id')
+    .where('id NOT IN (SELECT "userId" FROM public.alerts)')
+    .getQuery();
+
+  await con
+    .getRepository(Alerts)
+    .query(`INSERT INTO public.alerts ("userId") ${usersQuery}`);
```

Comment:
The default set in SQL are matching our `ALERTS_DEFAULTS`?

Reply:
I think it is better to stick with DB defaults (as we also set the default values there). Especially, our trigger is based on DB default values.

Reply:
I understand, I don't want to change it, mainly want to avoid there being discrepancies between `ALERT_DEFAULTS` and the tables defaults as we are now changing from 1 to the other.
Think the above one solves it, so will resolve this.